build_docker_images.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+set -euo pipefail
+
+# Script to generate Docker images for Stargate
+# Assumes that you have done a complete build so that all jars have been created, i.e.:
+#   mvn install -P dse
```

Comment:
I think we should always point to a wrapper and skip tests:

`./mvnw install -P dse -DskipTests=true`

Reply:
ok